codesigning/signing-worker/playbook.yml

```diff
@@ -0,0 +1,116 @@
+---
+
+- name: Set up everything needed on NCP-Inkwell
+  hosts: all
+  become: true
+  become_method: sudo
+  tasks:
+    - name: Install GCP SDK dependencies
+      ansible.builtin.apt:
+        update_cache: true
+        pkg:
+          - apt-transport-https
+          - ca-certificates
+          - gnupg
+          - curl
+
+    - name: Download the Google Cloud SDK package repository signing key
+      ansible.builtin.shell:
+        cmd: curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg
+        creates: /usr/share/keyrings/cloud.google.gpg
+
+    - name: Add Google Cloud SDK package repository source
+      ansible.builtin.apt_repository:
+        update_cache: true
+        filename: google-cloud-sdk.list
+        repo: "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main"
+
+    - name: Install packages
+      ansible.builtin.apt:
+        update_cache: true
+        pkg:
+          - python3
+          - python3-pexpect
+          - python3-requests
+          - wget
+          - vim-nox
+          - yubico-piv-tool
+          - libengine-pkcs11-openssl
+          - ykcs11
+          - libssl-dev
+          - google-cloud-sdk
+          - google-cloud-cli
+          - yubikey-manager
+
+    - name: Add bookworm-backports repository
+      ansible.builtin.apt_repository:
+        update_cache: true
+        repo: "deb http://deb.debian.org/debian {{ ansible_distribution_release }}-backports main"
+        filename: bookworm-backports.list
+
+    - name: Install osslsigncode from backports
+      ansible.builtin.apt:
+        update_cache: true
+        default_release: "{{ ansible_distribution_release }}-backports"
+        pkg:
+          # The backports version is needed because the version in bookworm has
+          # a critical bug in it that prevents it from working with our
```

Comment:
Is this bug something that's likely to be fixed soon/ever? It could be helpful to include a link to the bug, if possible, so we can check on it in the future

Reply:
Yes!  It was a bug introduced in osslsigncode 2.5 (which is the version in `debian:bookworm`) and was fixed in 2.6.  The version in `debian:bookworm-backports` is version 2.9, which is the latest released.  I've updated the comments here to reflect this in dfc275ae1